src/core/modules/NebulaMgr.cpp

```diff
@@ -1739,6 +1785,13 @@ void NebulaMgr::updateSkyCulture(const QString& skyCultureDir)
 			return;
 		}
 		loadDSONames(dsoNamesPath);
+		QString dsoDiscoveryPath = StelFileMgr::findFile("nebulae/" + setName + "/discovery.dat");
+		if (dsoDiscoveryPath.isEmpty())
+		{
+			qWarning().noquote() << "ERROR while loading deep-sky discovery data set " << setName;
+			return;
```

Comment:
not sure if return is the best way here here. There is an else branch below, but I don't see the context now. Maybe "continue"?

Reply:
Good point!